### PR TITLE
Bump target sdk version to 33

### PR DESCRIPTION
**Pull request by NUmeroAndDev**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- Updated target sdk version.
- Implemented notification permission.  

## Links
- https://developer.android.com/develop/ui/views/notifications/notification-permission

## Screenshot

- when allowed permission

https://user-images.githubusercontent.com/13705006/193743608-2a17f88f-0bb6-4657-98e1-52a9e0010c85.mp4

- when rejected permission

https://user-images.githubusercontent.com/13705006/193743444-c15aeb0b-3f58-42f7-9bf6-2e2b8160ed99.mp4

- show notification

https://user-images.githubusercontent.com/13705006/193749228-6545367d-f4b4-4676-b9e3-d2ad68a8c5d4.mp4

### CODE REVIEW

File: feature/about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt [hunk 1]
```diff
@@ -304,11 +307,18 @@ private fun ExternalServiceImage(
...
 private fun versionName(context: Context) = runCatching {
-    context.packageManager
-        .getPackageInfo(
+    val info = if (VERSION.SDK_INT >= VERSION_CODES.TIRAMISU) {
+        context.packageManager.getPackageInfo(
+            context.packageName,
+            PackageInfoFlags.of(0)
+        )
+    } else {
+        context.packageManager.getPackageInfo(
```
Comment (NUmeroAndDev) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt:316]: ref: https://developer.android.com/reference/android/content/pm/PackageManager#getPackageInfo(java.lang.String,%20int)

**Comment by takahirom** [comment]
Can you check session alarm?👀

**Comment by NUmeroAndDev** [comment]
I forgot to implement notification permission. 😅 
I thinks to implement display it when the app launched.

**Comment by NUmeroAndDev** [comment]
Added missing notification permission implementation. 🙏

**Pull request merged**